content/html/concepts/attributes/attributes.md

```diff
@@ -13,22 +13,40 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Attributes can be added to the opening tag of an HTML element to change its default behavior or provide additional data about it.
+HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
+Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 
 
-Attributes consist of a name and a value in the format `name="value"`:
+## Syntax 
 
-```pseudo
-<tag name="value"></tag>
+The syntax for adding attributes to an HTML element is as follows: 
+
+ ```html 
+ <tagname attribute="value"></tagname> 
+ ```
+
+> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.
+ 
+## Examples 
+
+ ### Image Attributes
+
+Using an  [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the particular element, once specified, enable precise control over the end result.
+
+```html
+<img src="source_of_image.png" width="500" height="600" />
 ```
 
-HTML elements can have any number of attributes.
+![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/changing-height-attribute-of-html-element.gif)
```

Comment:
```suggestion
![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/Changing-height-attribute-of-html-element.gif)
```

Reply:
Hi  @caupolicandiaz , I have committed the changes from your review, Unfortunately, something is wrong with my 
local repo of Codecademy docs, and it's missing the documents that I have committed not 100% sure what happened therefore I can't or don't seem to understand how to fix the capital letter in the directory expect to start from scratch in new repo. Could you assist me in it or let the name be like so?

Reply:
The file doesn't need to be changed. The link did not match the existing case, so the change put them in alignment